src/compiler/cicc.rs

```diff
@@ -329,23 +317,22 @@ pub fn generate_compile_commands(
 }
 
 ArgData! { pub
-    Output(PathBuf),
-    PassThrough(OsString),
-    UnhashedFlag,
+    ExtraOutput(PathBuf),
```

Comment:
I renamed this from `UnhashedOutput -> ExtraOutput` to reflect that these argument types are now hashed.